Week3/More Javascript/script.js

```diff
@@ -0,0 +1,148 @@
+//1
+
+function info(a, b, c) {
```

Comment:
The name `info` isn't really descriptive of what this function will do. Do you know a better name for this function?

Reply:
Got it. :)